specification/oracle/Oracle.Database/tspconfig.yaml

```diff
@@ -0,0 +1,15 @@
+emit:
+  - "@azure-tools/typespec-autorest"
```

Comment:
it didn't enable the sdk emitter here because the mpg has not ready yet?

Reply:
With this PR we are focusing only on API public preview. All changes for SDK generation will come in separate PR.